iceoryx_hoofs/test/moduletests/test_allocator.cpp

```diff
@@ -76,7 +78,10 @@ TEST_F(Allocator_Test, allocateTooMuchSingleElement)
     ::testing::Test::RecordProperty("TEST_ID", "9deed5c0-19d8-4469-a5c3-f185d4d881f1");
     iox::posix::Allocator sut(memory, memorySize);
     std::set_terminate([]() { std::cout << "", std::abort(); });
+    // NOLINTBEGIN(hicpp-avoid-goto, cppcoreguidelines-avoid-goto, cert-err33-c, cppcoreguidelines-pro-type-vararg,
+    // hiccpp-vararg) death test
     EXPECT_DEATH({ sut.allocate(memorySize + 1, MEMORY_ALIGNMENT); }, ".*");
+    // NOLINTEND
```

Comment:
```suggestion
    // todo #1196 remove EXPECT_DEATH
    // NOLINTNEXTLINE(hicpp-avoid-goto, cppcoreguidelines-avoid-goto, cert-err33-c, cppcoreguidelines-pro-type-vararg, hiccpp-vararg)
    EXPECT_DEATH({ sut.allocate(memorySize + 1, MEMORY_ALIGNMENT); }, ".*");
```

Reply:
Changed for all death tests in this file.